internal/common/client_test.go

```diff
@@ -0,0 +1,44 @@
+package common
+
+import (
+	"github.com/SOMAS2020/SOMAS2020/internal/common/rules"
+	"gonum.org/v1/gonum/mat"
+	"testing"
+)
+
+// TestBasicRuleEvaluatorPositive Checks whether rule we expect to evaluate as true actually evaluates as such
+func TestBasicRuleEvaluatorPositive(t *testing.T) {
+	result, err := BasicRuleEvaluator("Kinda Complicated Rule")
```

Comment:
Hmm... this test relies on the registration of `"Kinda Complicated Rule"`. which we might change. It's nice to have self-contained unit tests that don't rely on other (possibly side-effecty) code.

w.r.t. this file we could have the logic for `registerTestRule` contained in the `TestBasicRuleEvaluatorNegative` function as well. 

Thanks for the unit tests though! Tests are always hard to write!

Reply:
Also this might only be important in the final PR, but having them self-contained also means you can have one const. variable referring to supposed constants like the rule name :)